nhaystack-wb/src/nhaystack/ui/Row.java

```diff
@@ -306,11 +306,11 @@ private static BWbFieldEditor makeRefFE(
             }
             if(dict != null)
             {
-                // workaround for weird problem with axSlotPath
-                if (dict.has("axSlotPath"))
-                    ord = BOrd.make("station:|" + dict.getStr("axSlotPath"));
+                // workaround for weird problem with n4SlotPath
+                if (dict.has("n4SlotPath"))

```

Comment:
No problems, the change was made because I recall the suggestion made that we should use `n4` as a prefix in preference to `ax`.  I wasn't aware of any production systems using this feature so didn't anticipate breakage.

I've added logic that falls back to using `axSlotPath` in the situation where `n4SlotPath` is absent, so it should handle either case.